src/test/kotlin/uk/gov/dluhc/emsintegrationapi/rest/UpdatePendingRegisterCheckIntegrationTest.kt

```diff
@@ -930,25 +924,13 @@ internal class UpdatePendingRegisterCheckIntegrationTest : IntegrationTest() {
         assertRequestIsAudited(actualRegisterResultData, requestId, createdAtFromRequest, gssCode, matchCount)
 
         assertMessageSubmittedToSqs(
-            queueUrl = localStackContainerSettings.mappedQueueUrlConfirmRegisterCheckResult,
+            queueUrl = localStackContainerSettings.mappedQueueUrlRegisterCheckResultResponse,
             expectedMessageContentSentToVca,
         )
     }
 
-    @ParameterizedTest
-    @CsvSource(
-        value = [
-            "VOTER_CARD, VOTER_MINUS_CARD,",
-            "POSTAL_VOTE, POSTAL_MINUS_VOTE,",
-            "PROXY_VOTE, PROXY_MINUS_VOTE,",
-            "OVERSEAS_VOTE, OVERSEAS_MINUS_VOTE,",
-            "APPLICATIONS_API, APPLICATIONS_MINUS_API",
-        ],
-    )
-    fun `should submit the different service result messages to the correct queues`(
-        sourceTypeEntity: SourceTypeEntity,
-        sourceType: SourceType,
-    ) {
+    @Test
+    fun `should submit the different service result messages to the correct queues`() {
```

Comment:
```suggestion
    fun `should submit the result messages to the correct queue`() {
```

How is this test different from the one above?

Reply:
Ah nice I missed this! Thanks